src/cljs/webchange/ui_framework/components/menu/index.cljs

```diff
@@ -30,7 +30,9 @@
   [menu-button list-position]
   (let [{:keys [x y width height]} (get-bounding-rect menu-button)]
     (case list-position
-      "bottom" {:top  (+ y height)
+      "bottom" {:top  (if (> 371 y)
```

Comment:
what `371` and `191` is? If it's not computable then set a variable with conscious name

Reply:
Done

Reply:
370 is the max-space after which the wc-menu modal will open on top of the phrase, while 190 is the fixed height of wc-menu modal